urllib3/contrib/pyopenssl.py

```diff
@@ -217,7 +217,10 @@ def shutdown(self):
 
     def close(self):
         if self._makefile_refs < 1:
-            return self.connection.close()
+            try:
+                return self.connection.close()
+            except OpenSSL.SSL.Error:
+                return
```

Comment:
I used "return" rather than "pass" so that you don't need to look any further in the function to see that falling through does nothing, but it's just a matter of style.

As for return value, there were already circumstances in which `None` was returned implicitly. The only potentially useful return value is that from `self.connection.close()`, but I don't know if that ever returns anything, nor if anyone ever checks it.